gsoc2018-bharat/src/WikiDetector.py

```diff
@@ -0,0 +1,167 @@
+import os
+import re
+import sys
+import time
+import tempfile
+from joblib import Parallel, delayed
+from urllib.parse import unquote
+from collections import Counter
+
+def upcase_first_letter(s):
```

Comment:
isn't this just `s.capitalize()` ? note that your method would trown an Exception for empty strings.

Reply:
The purpose was also to replace spaces with underscore. I will make the changes so that it works inline itself.